scientific_library/tvb/simulator/models/wong_wang_exc_inh.py

```diff
@@ -182,9 +182,15 @@ class ReducedWongWangExcInh(ModelNumbaDfun):
         domain=Range(lo=0.0, hi=1.0, step=0.001),
         doc="""[nA]. Effective external input""")
 
+    I_ext = NArray(
+        label=":math:`I_{ext}`",
+        default=numpy.array([0.0, ]),
+        domain=Range(lo=0.0, hi=1.0, step=0.001),
+        doc="""[nA]. Effective external stimulus input""")
+
     G = NArray(
         label=":math:`G`",
-        default=numpy.array([2.0, ]),
+        default=numpy.array([20.0, ]),
```

Comment:
If this model is present in a current TVB release, please do not change the default parameter values; a jump from 2 to 20 is pretty big.

Reply:
ok